wiki/Storyboard/Scripting/osu!_File_Toggles/de.md

```diff
@@ -1,11 +1,11 @@
-# Storyboard Scripting - Schalter
+# Storyboard-Einstellungen in .osu-Dateien
```

Comment:
Is there a precedent of this title format? I don't know if I agree with translating `scripting` as `Einstellungen`.

Reply:
So, the English title is `Storyboard .osu file toggles` and if I understand it correctly, this article lists entries that can be added to `.osu` or `.osb` files to *configure* stuff related to storyboards.

Originally, this article translated `toggles` with `Schalter`, but I think `Einstellungen` fits better in this context.

As a suggestion, I could agree to use `Einstellungen beim Storyboard-Scripting in .osu-Dateien` if that is what you are going for.